### PR TITLE
fix(input): fix input divergences from style guide

### PR DESCRIPTION
**Pull request by pedrohmorais**
## Description
https://jirasoftware.catho.com.br/browse/QTM-61

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [x] Unit tests (yarn test:components)
- [x] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [x] Code review

### Browsers review
- [x] Layout review
  - [x] Edge
  - [x] IE 11
  - [x] Firefox
  - [x] Chrome
  - [x] Safari
  - [x] Mobile
- [ ] Ux review validation

### CODE REVIEW

**Comment by rapahaeru** [comment]
Question: do we need the description label?
I think it can be useful, but its not in the style guide.

What do you think?

image

**Comment by rapahaeru** [comment]
We need to implement a new props to flexibilize the size of the icon.
image

this is new feature.

**Comment by rapahaeru** [comment]
I think we need a new boolean prop to set a disable input.
Wha do you think?

image

**Comment by rapahaeru** [comment]
In mobile, iPhone, the search type of input stills the apple format.
This problem only happens in search type.

image

File: components/Input/Input.jsx [hunk 1]
```diff
@@ -34,32 +34,27 @@ const InputSearchIcon = styled(InputIcon).attrs({ name: 'search' })`
   pointer-events: none;
   ${({
     theme: {
-      spacing: { small },
+      spacing: { medium },
     },
-  }) => `left: ${small}px;`}
+  }) => `left: ${medium}px;`}
```
[the remark this answers is not in the document]
Comment (pedrohmorais) [reply components/Input/Input.jsx]: Fixed in b87991e41790a569dddfa4fb51e79b4d3d352ff0

**Pull request merged**